translations/de/info/task_description.html

```diff
@@ -0,0 +1,62 @@
+<img class="for_info_only half" title="X" src="{{MEDIA}}X.png" alt="X"
+     style="float: left"/>
+
+
+<p>
+    Versuchen wir's mit etwas Sortieren. Hier ist ein Array (Datenfeld) mit den spezifischen Regeln. 
```

Comment:
> ...  ein Array `(Datenfeld)` ...

this term is definitely wrong, I recommend to use either `Liste` or `Sequenz`, but basically it needs no translation
as Duden knows the word "Array" so it can be used in german as well